apps/src/templates/CertificateShare.jsx

```diff
@@ -1,20 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import i18n from '@cdo/locale';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import {TwoColumnActionBlock} from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
+import styleConstants from '../styleConstants';
 
 export default function CertificateShare(props) {
```

Comment:
Potential refactor: destructure the props like `CertificateShare({announcement, printUrl, imageUrl})` and use the prop names directly instead of reading them off of `props`